django/contrib/admin/options.py

```diff
@@ -1744,6 +1745,9 @@ def __init__(self, parent_model, admin_site):
     def media(self):
         extra = '' if settings.DEBUG else '.min'
         js = ['jquery%s.js' % extra, 'jquery.init.js', 'inlines%s.js' % extra]
+        # TODO: Was removed in master, modified in patch.
```

Comment:
The idea is that `InlineModelAdmin` is always used with a `ModelAdmin`, so we don't need to include the files again.